src/System.Data.OleDb/tests/OleDbCommandBuilderTests.cs

```diff
@@ -154,7 +155,24 @@ private void RunTest(Action<OleDbCommand, string> testAction, [CallerMemberName]
                     Firstname NVARCHAR(5),
                     Lastname NVARCHAR(40), 
                     Nickname NVARCHAR(30))";
-            command.ExecuteNonQuery();
+            try
+            { 
+                command.ExecuteNonQuery();
+            }
+            catch (SEHException sehEx)
+            {
+                Console.WriteLine($"Code: {sehEx.ErrorCode}");
+                Exception baseException = sehEx.GetBaseException();
+                if (baseException != null)
```

Comment:
GetBaseException should never return null.  While technically an exception could override it to do so, that would be a badly implemented exception.